### PR TITLE
(docs):created documentation for InstantSearch

### PR DESCRIPTION
**Pull request by ThaisChaves**
**Related issue**

**Summary**

**Screenshots**

**Checklist**

- [ ] I've read and understood [contributing guidelines](https://github.com/ecomplus/storefront/blob/master/CONTRIBUTING.md);
- [ ] If changing UI, I've tested on most common viewports, desktop and mobile devices;

**A picture of a cute animal**

### CODE REVIEW

File: @ecomplus/storefront-components/docs/AInstantSearch.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+Reduces the vertical size in a way that renders the component using less space. Ideal for use on a pop-up screen, for example.
+
+```js
+productCardProps: {
+  type: Object,
+  default () {
+  return {
+    isSmall: true
+     }
```
Comment (leomp12) [inline @ecomplus/storefront-components/docs/AInstantSearch.md:97]:
```suggestion
  }
},
```

Comment (leomp12) [reply @ecomplus/storefront-components/docs/AInstantSearch.md:97]: Tava faltando uma `}` aqui

File: @ecomplus/storefront-components/docs/AInstantSearch.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+### term
+Standard text that will start the search
+For example: If you leave it set, `:term.sync= "chair";` the search will open with the search for this product. It is the 'keyword' of the search.
+
+ ```js
+term: {
+  type: String,
+  default: ''
+    }
```
Comment (leomp12) [inline @ecomplus/storefront-components/docs/AInstantSearch.md:48]:
```suggestion
},
```

Comment (leomp12) [reply @ecomplus/storefront-components/docs/AInstantSearch.md:48]: Fixando identação

File: @ecomplus/storefront-components/docs/AInstantSearch.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+```
+
+### pageSize
+
+The number of products returned from the search. This is a way to limit the search result and show only some of the products. Note that if the number of products is more that the products that can fit in the autocomplete results window then a scroll bar will appear.
+
+```js
+pageSize: {
+  type: Number,
+  default: 8}
```
Comment (leomp12) [inline @ecomplus/storefront-components/docs/AInstantSearch.md:69]:
```suggestion
  default: 8
},
```

File: @ecomplus/storefront-components/docs/AInstantSearch.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+#### All props
+
+```vue
+<InstantSearch
+  :term.sync="term"
+  :isVisible="true"
+  :pageSize="4"
+  :autoFixScore="0.80"
+  :searchEngineProp ?????
```
Comment (leomp12) [inline @ecomplus/storefront-components/docs/AInstantSearch.md:33]:
```suggestion
  :searchEngineProps="{}"
```

File: @ecomplus/storefront-components/docs/AInstantSearch.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+```
+
+#### All props
+
+```vue
+<InstantSearch
+  :term.sync="term"
+  :isVisible="true"
+  :pageSize="4"
+  :autoFixScore="0.80"
```
Comment (leomp12) [inline @ecomplus/storefront-components/docs/AInstantSearch.md:32]:
```suggestion
  :autoFixScore="0.8"
```

File: @ecomplus/storefront-components/docs/AInstantSearch.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+  :is-visible.sync="isVisible"
+/>
+```
+
+#### All props
+
+```vue
+<InstantSearch
+  :term.sync="term"
+  :isVisible="true"
```
Comment (leomp12) [inline @ecomplus/storefront-components/docs/AInstantSearch.md:30]:
```suggestion
  :isVisible.sync="isVisible"
```

File: @ecomplus/storefront-components/docs/AInstantSearch.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+  :pageSize="4"
+  :autoFixScore="0.80"
+  :searchEngineProp ?????
+  :productCardProps ?????
```
Comment (leomp12) [inline @ecomplus/storefront-components/docs/AInstantSearch.md:34]:
```suggestion
  :productCardProps="{
    isSmall: false,
    headingTag: 'h4'
  }"
```

File: @ecomplus/storefront-components/docs/AInstantSearch.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+
+### isVisible
+
+Defines whether the search window will be opened or not when entering the page.
+
+```js
+isVisible: {
+  type: Boolean,
+  default: true
+    }
```
Comment (leomp12) [inline @ecomplus/storefront-components/docs/AInstantSearch.md:64]:
```suggestion
}
```

File: @ecomplus/storefront-components/docs/AInstantSearch.md [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+term: {
+  type: String,
+  default: ''
+},
+```
+
+### isVisible
+
+Defines whether the search window will be opened or not when entering the page.
+
```
Comment (leomp12) [inline @ecomplus/storefront-components/docs/AInstantSearch.md:59]:
```suggestion

> May be used with `.sync` modifier.

```

**Comment by ThaisChaves** [comment]
Pode colocar os exemplos. Estou fazendo a documentação do Bootstrap

**Pull request merged**